docs/proposals/scheduledjob.md

```diff
@@ -0,0 +1,132 @@
+# ScheduledJob Controller
```

Comment:
Periodic suffers some of the same usability concerns.  I do think API
objects should be precise in their meaning, but designed for end users, not
systems researchers.

On Fri, Jul 31, 2015 at 1:02 PM, Timothy St. Clair <notifications@github.com
> wrote:

> In docs/proposals/scheduledjob.md
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/11980#discussion_r35992880>
> :
>
> > @@ -0,0 +1,132 @@
> > +# ScheduledJob Controller
>
> Proposal was around (diurnal, periodic, etc.) There may be a couple more.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/11980/files#r35992880>
> .
>



-- 
Clayton Coleman | Lead Engineer, OpenShift